### DISCUSSION TITLE
1.17.5 version virus detected

### DISCUSSION THREAD

**Discussion by epaluy** [maximmax42/Discord-CustomRP#202] [General]
Hey is it just me or is the newest version 1.17.5 says there is a virus detected when you download from website, github or even update from previous version

**Comment by maximmax42** [comment]
There's no virus. It's a false positive. It happens every damn time I release a new version, and I have to message antivirus devs to rescan the app manually.
Where did you get a warning from? Windows Defender?

**Reply by epaluy** [reply]
Sry for late response, I’m not sure where it came from I don’t have any anti virus apps, it just popped and alert message saying it’s a virus when I tried to run it. Maybe it’s reputation based protection, im not to sure

**Reply by maximmax42** [reply]
Could be reputation based, yeah.
